CMakeLists.txt

```diff
@@ -282,11 +282,18 @@ pkg_check_modules(XCB_COMPOSITE REQUIRED xcb-composite)
 pkg_check_modules(XCB_XFIXES REQUIRED xcb-xfixes)
 pkg_check_modules(XCB_RENDER REQUIRED xcb-render)
 pkg_check_modules(X11_XCURSOR REQUIRED xcursor)
+pkg_check_modules(DRM REQUIRED libdrm)
```

Comment:
Will requiring drm cause problems for UBports builds targetting libhybris/android? @mariogrip

Reply:
As I mentioned in the first commit message, we already [unconditionally require libdrm](https://github.com/MirServer/mir/blob/34f92104efea90aaa79f90d9acd5b9f638f3d21f/src/server/console/minimal_console_services.cpp#L32
); we just don't unconditionally check for it.

Reply:
Ack. Makes sense.